pkg/client/unversioned/portforward/portforward_test.go

```diff
@@ -206,7 +206,7 @@ func TestGetListener(t *testing.T) {
 }
 
 // fakePortForwarder simulates port forwarding for testing. It implements
-// kubeletserver.PortForwarder.
+// portforward.PortForwarder.
```

Comment:
replacing it with `var _ portforward.PortForwarder = fakePortForwarder` should be much saner since this a contract